components/CaseTable.tsx

```diff
@@ -9,21 +12,6 @@ interface ICaseTableProps {
     synapseAtlases: Atlas[];
 }
 
-// we need sort by participant id which takes the form HTA[integer]_[integer]
-const sortFunction = (rows: any[], field: string, direction: any) => {
```

Comment:
Same as BiospecimenTable, this was only working for a single field only in ascending direction.